kitsune/sumo/management/commands/merge.py

```diff
@@ -0,0 +1,57 @@
+from kitsune.settings import SUMO_LANGUAGES
+from django.core.management.base import BaseCommand
+from babel.messages.frontend import CommandLineInterface
+
+
+class Command(BaseCommand):
+    """
+    Merge will merge the .pot files into the existing l10n files,
+    which exist in locale (if you have the repo downloaded).
+
+    """
+
+    help = "Merges the .pot files into the existing l10n files."
+
+    def handle(self, *args, **options):
+        no_repo_locales = []
+        for locale in SUMO_LANGUAGES:
```

Comment:
You can exclude the "en-US" locale and "xx" test locale.
```suggestion
        for locale in settings.SUMO_LANGUAGES:
            if locale in ("en-US", "xx"):
                continue
```

Reply:
Agreed.